homeassistant/components/epson/media_player.py

```diff
@@ -59,17 +60,20 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_PORT, default=80): cv.port,
```

Comment:
We shouldn't change the legacy schema. That would break users that import the old config. If we don't want to save the port in the config entry data, we can pop the port item from the imported data before creating the entry.